client/src/components/ExportModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { Dialog, Transition } from '@headlessui/react';
+import { useState, Fragment } from 'react';
+
+import Button from './Button';
+
+const ExportModal = ({
+  children,
+  exportUrl,
+  exportFilename,
+}: {
+  children: React.ReactNode;
+  exportUrl: string;
+  exportFilename: string;
+}) => {
```

Comment:
Please define an interface that extends `React.HTMLAttributes<HTMLDivElement>` and then spread `{...rest}` on the main div. You don't have to include `children` in the interface since it's already included in the HTML attributes, but add `className` and destructure that and set it on the div as well.

See this for reference:
https://github.com/davidpak/SmartNote/pull/43/files